packages/@aws-cdk/aws-cloudfront/test/test.basic.ts

```diff
@@ -475,4 +479,266 @@ export = {
     test.done();
   },
 
+  'viewerCertificate': {
+    'acmCertificate': {
+      'base usage'(test: Test) {
+        const stack = new cdk.Stack();
+        const sourceBucket = new s3.Bucket(stack, 'Bucket');
+
+        const certificate = new certificatemanager.Certificate(stack, 'cert', {
+          domainName: 'example.com',
+        });
+
+        new CloudFrontWebDistribution(stack, 'AnAmazingWebsiteProbably', {
```

Comment:
Probably!

Reply:
Yeah, I copied it from a previous test, but we should be testing the amazing-ness of the website. Any suggestion on a threshold? 😄